rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -928,12 +929,17 @@ def process(self, message: Message, **kwargs: Any) -> None:
 
             message.set(ENTITIES, entities, add_to_output=True)
 
+        if out and DIAGNOSTIC_DATA in out:
+            message.add_diagnostic_data(
+                self.unique_name,
+                rasa.utils.tensorflow.numpy.values_to_numpy(out.get(DIAGNOSTIC_DATA)),
```

Comment:
seems the import for `rasa.utils.tensorflow.numpy` is missing (I get a warning in pycharm)